### DISCUSSION TITLE
Idea: Public chat room to discuss responses to the polycrisis, connect on a more regular basis & network

### DISCUSSION THREAD

**Discussion by nathenf** [life-itself/community#180] [General]
In our recent survey, 68% of respondents wanted great networking and gathering opportunities and 42% wanted some form of mentorship. Both of these could be provided through an active chat channel of mutual support and open dialogue. 

### Brainstorm

- The greatest downfall of chat channels is empty room syndrome 
- Building the habit of opening up a new app or website to enter a chat is a huge barrier to entry
- On the flipside a chat channel with too many people in the room can feel less "safe" to be vulnerable and makes it difficult to maintain mutual support 
- We could use an app like Whatsapp, which people already regularly use and have a cohort based introduction method to maintain a solid container and encourage participants to decide whether they wish to stay in (actively) the group. The previous cohort takes on the role of mentoring the new cohort. The "Elders" therefore maintain the principles of mutual support, authentic relating and non-violence, by leading the way and supporting and encouraging the new cohort. 

Just some initial ideas, what do you think?

**Comment by rufuspollock** [comment]
@nathenf i think a great idea to try this out. I think there is a distinction between something basic to get things rolling and ensuring something high quality. i think we could already try something basic here e.g. using whatsapp.

Some additional comments. 

> The greatest downfall of chat channels is empty room syndrome

At the very least we have "core" team who are already using chat quite a bit. some of this could migrate if we made an effort.

> Building the habit of opening up a new app or website to enter a chat is a huge barrier to entry

if we can incorporate into an existing chat system this may help.
